<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecalotPersoMainDialog</class>
 <widget class="QDialog" name="SecalotPersoMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>357</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Secalot Bitcoin Personalization</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QGroupBox" name="PINCodeGroupBox">
     <property name="title">
      <string>New PIN code</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="PINCodeLabel">
        <property name="text">
         <string>Enter the new PIN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PINCodeLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxLength">
         <number>32</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Enter your new wallet PIN code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RepeatPINCodeLabel">
        <property name="text">
         <string>Repeat the new PIN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RepeatPINCodeLineEdit">
        <property name="maxLength">
         <number>32</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Repeat your new wallet PIN code</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="NewOrRestoreGroupBox">
     <property name="title">
      <string>Wallet</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="CreateNewWalletRadioButton">
        <property name="text">
         <string>New wallet</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="RestoreSeedLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter a hexadecimal or a BIP39 mnemonic seed</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="RestoreWalletRadioButton">
        <property name="text">
         <string>Restore wallet</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Here you can personalize the bitcoin application of your Secalot device. You can either create a new wallet, or restore a wallet from a known seed.
Plus you need to specify the PIN code for the newly created wallet. PIN length should be between 4 and 32 characters.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CreateWalletPushButton">
       <property name="text">
        <string>Create Wallet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Secalot bitcoin personalization</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CreateNewWalletRadioButton</tabstop>
  <tabstop>RestoreWalletRadioButton</tabstop>
  <tabstop>RestoreSeedLineEdit</tabstop>
  <tabstop>PINCodeLineEdit</tabstop>
  <tabstop>RepeatPINCodeLineEdit</tabstop>
  <tabstop>CreateWalletPushButton</tabstop>
  <tabstop>CancelPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>CancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SecalotPersoMainDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RestoreWalletRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>RestoreSeedLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
